uiowa/uiowa
# Feature: Layout Builder default content

Resolves: #1925 

See this comment for what needs to be done to finish the work on this: https://github.com/uiowa/uiowa/blob/1277ebbc13d305a1cfea42a7c7c937e14f249348/docroot/profiles/custom/sitenow/content/block_content/0c0c1f36-3804-48b0-b384-6284eed8c67e.json#L12

# To test
* blt drupal:install --site=cogscilang.grad.uiowa.edu
* Confirm that the default content installs and that there are no visible issues with it
